person.rb

```diff
@@ -0,0 +1,28 @@
+class Person
+  @count = 0
+
+  def self.count
+    @@count
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(name: 'Unknown', age: nil, parent_permission: true)
+    @count += 1
```

Comment:
- [ ] Great work so far 👍 but there are some changes that need to be done here 📝

**1.** First the age variable is supposed to be given by only user, not the app itself 🤔 so kindly remove the default value from the age variable to make your variables according to the project requirements 👍

**2.** It would be nice if you used an equal `(=)` sign instead of the colon `(:)` sign for the default value of the name because whenever you pass a string as a default value then you use the equal sign 🤔

After making the above changes your initialize will look like the below ⬇️
```ruby
def initialize(age, name= 'Unknown', parent_permission: true)
```
**3.** After making the above changes it would be nice if you also correct the **student.rb** and **teacher.rb** files 👍